api/PerformancePaintTiming.json

```
@@ -14,7 +14,7 @@
             "version_added": "≤79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "84"
           },
           "firefox_android": {
             "version_added": null
```

Looks like this also appeared in Android

```suggestion
            "version_added": "84"
```